src/static/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
This file should already be minified.
If you can't find minified file - just minify it using some online css minifier